test/unit/app/controllers/permissions/permissions-controller-test.js

```diff
@@ -934,24 +937,24 @@ describe('permissions controller', function () {
     })
 
     it('should emit notification for multiple connected domains', async function () {
-      identities[ACCOUNT_ARRAYS.a[1]] = { lastSelected: 1000 }
+      identities[ACCOUNTS.a.permitted[1]] = { lastSelected: 1000 }
       assert(preferences.subscribe.calledOnce)
       assert(preferences.subscribe.firstCall.args.length === 1)
       const onPreferencesUpdate = preferences.subscribe.firstCall.args[0]
 
-      await onPreferencesUpdate({ selectedAddress: ACCOUNT_ARRAYS.a[1] })
+      await onPreferencesUpdate({ selectedAddress: ACCOUNTS.a.permitted[1] })
 
-      const accountsWithoutFirst = ACCOUNT_ARRAYS.a
-        .filter((account) => account !== ACCOUNT_ARRAYS.a[1])
-      const expectedAccounts = [ ACCOUNT_ARRAYS.a[1], ...accountsWithoutFirst ]
+      // const accountsWithoutFirst = ACCOUNTS.a.permitted
+      //   .filter((account) => account !== ACCOUNTS.a.permitted[1])
+      // const expectedAccounts = [ ACCOUNTS.a.permitted[1], ...accountsWithoutFirst ]
```

Comment:
I take it we don't need these?
```suggestion
```

Reply:
In the event that the user has provided a future 'access all accounts' permission, we might.

Reply:
Let's delete these lines—we can restore them from history if we need them

Reply:
This was done